packages/webapp/pages/posts/[id]/edit.tsx

```diff
@@ -104,7 +105,19 @@ function EditPost(): ReactElement {
         isForbidden={!isVerified || !squad || !canEdit}
       >
         <WritePostHeader isEdit />
-        <WriteFreeformContent className="py-6 px-4" />
+
+        {post?.type === PostType.Share ? (
```

Comment:
I think ideally we use a completely different mutation for editing actually.
Like we do for EditPost it's a better failsafe way.

Not 100% sure how it's tackled in the freeform one now, but maybe we can look at that logic and re-apply?